lib/modules/manager/gradle-wrapper/artifacts.spec.ts

```diff
@@ -275,4 +275,51 @@ describe('modules/manager/gradle-wrapper/artifacts', () => {
     ]);
     expect(execSnapshots).toBeEmptyArray();
   });
+
+  it('handles gradle-wrapper in subdirectory', async () => {
+    const execSnapshots = mockExecAll();
+    git.getRepoStatus.mockResolvedValue({
+      modified: [
+        'sub/gradle/wrapper/gradle-wrapper.properties',
+        'sub/gradlew',
+        'sub/gradlew.bat',
+      ],
+    } as StatusResult);
+
+    const res = await updateArtifacts({
+      packageFileName: 'sub/gradle/wrapper/gradle-wrapper.properties',
+      updatedDeps: [],
+      newPackageFileContent: Fixtures.get(
+        'expectedFiles/gradle/wrapper/gradle-wrapper.properties'
+      ),
+      config: { ...config, newValue: '6.3' },
+    });
+
+    expect(res).toEqual(
```

Comment:
don't use map, check other tests

Reply:
I copied this 1:1 from your refactored tests! 

https://github.com/renovatebot/renovate/blob/a657c3476e7a6c78034147ae04c71ae81b88bd17/lib/modules/manager/gradle-wrapper/artifacts.spec.ts#L93-L100

Reply:
Apart from that, I don't see the problem with map

Reply:
ok, forgot to refactor that test 😅